README.md

```diff
@@ -109,7 +115,7 @@ hash, obj = s.traverse_base(test_base)
 # send it to the server
 objCreate = client.object.create(stream_id="stream id", objects=[obj])
 
-received_base = client.object.get(hash)
+received_base = client.object.get(hash, obj["id"])
```

Comment:
good spot that the args for this are wrong! however, this should actually be `object.get("stream id", obj["id")` or `object.get("stream id", hash)` as the signature is  `get(self, stream_id: str, object_id: str)`

Reply:
Sorry for late answer. I got busy on other stuff. I thought it was good because it did not raise any exception but it actually return an exception without raising it. I did not expect this.

I think it is good now but I cannot test it correctly as I have not set up a speckle-server at the moment.

Reply:
no problem cyril, thanks for this fix! lgtm 👍